srcpkgs/latexmk/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'latexmk'
+pkgname=latexmk
+version=4.76
+revision=1
+_version_without_dot="${version/./}"
+wrksrc="${pkgname}"
+depends="perl"
+short_desc="Perl script for running LaTeX the correct number of times"
+maintainer="Tung Anh Vu <vu.tunganh96@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://personal.psu.edu/~jcc8/software/latexmk"
+distfiles="${homepage}/${pkgname}-${_version_without_dot}.zip"
+checksum=9abb87da4666d6078c9006dbb32dc2a7cb3996fc2f76b0c6c5c73311a50ac89e
+
```

Comment:
Possibly, but I'm currently too busy to address this. I will try to make time by the end of the week.

Reply:
You were right. I added `texlive` as a conflict.

However, `/usr/bin/latexmk` installed by texlive is only a symlink to a directory with TeX related scripts. Maybe there's a way to avoid installing this symlink if latexmk is installed?

Reply:
Maybe alternatives, but if we can stand 4.70b, it would be easier to pull latexmk from texlive as subpackage.

Reply:
Looks fine. Revision needs to be increased to build it in repo.

Reply:
Please reuse this one and do not close.

Reply:
Ok, that sounds good as well.